Description: Do not use memcache standard port
 Because of the way we run tests in Debian, it's advised not to use the
 standard port of memcached, so we can run another (test specific) instance
 of memcached.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2018-09-06

Index: python-memcache/tests/test_memcache.py
===================================================================
--- python-memcache.orig/tests/test_memcache.py
+++ python-memcache/tests/test_memcache.py
@@ -30,7 +30,7 @@ class FooStruct(object):
 class TestMemcache(unittest.TestCase):
     def setUp(self):
         # TODO(): unix socket server stuff
-        servers = ["127.0.0.1:11211"]
+        servers = ["127.0.0.1:11212"]
         self.mc = Client(servers, debug=1)
 
     def tearDown(self):
@@ -257,7 +257,7 @@ class TestMemcache(unittest.TestCase):
 class TestMemcacheEncoder(unittest.TestCase):
     def setUp(self):
         # TODO(): unix socket server stuff
-        servers = ["127.0.0.1:11211"]
+        servers = ["127.0.0.1:11212"]
         self.mc = Client(servers, debug=1, key_encoder=self.encoder)
 
     def tearDown(self):
